docs/rules/valid-jsdoc.md

```diff
@@ -290,6 +290,7 @@ function greet(name) {
 
 ### requireReturnType
 
+If you use https://www.npmjs.com/package/esdoc-flow-plugin, make a copy for all param types not needed. 
```

Comment:
If you use flowtype and esdoc + esdoc-flow-plugin or documentation.js, documentation of the types will be automatic, thus no need to specify return type manually.